docs/contributions-flow.md

```diff
@@ -0,0 +1,52 @@
+<!---
+Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# SwG Contributions Flow
+
+This flow allows the publication site to display a flow to contribute to
+the publisher. See [Subscriptions APIs](./core-apis.md).
+
+*Important!* Please ensure you set up the `setOnSubscribeResponse` on any page where you accept purchases, not just before you call `subscribe` or `showContributions`. SwG client ensures it can recover contributions even when browsers unload pages. See [Subscribe flow](./subscribe-flow.md) for more details.
```

Comment:
This is confusing. I personally don't understand exactly what it's referring to and expect that an outside developer would have a hard time as well.

Reply:
This probably belongs much lower in the doc

Reply:
I would leave this as is for now to be consistent with Offers flow.

We could discuss this how we could improve the docs overall.

Reply:
I would move this to the bottom and create an issue to improve the Offers flow to do the same

Reply:
Done.